Lib/test/test_pyrepl/test_windows_console.py

```diff
@@ -0,0 +1,331 @@
+import itertools
+import sys
+import unittest
+from _pyrepl.console import Event, Console
+from _pyrepl.windows_console import (
+    MOVE_LEFT,
+    MOVE_RIGHT,
+    MOVE_UP,
+    MOVE_DOWN,
+    ERASE_IN_LINE,
+)
+from functools import partial
+from typing import Iterable
+from unittest import TestCase, main
+from unittest.mock import MagicMock, call, patch, ANY
+
+from .support import handle_all_events, code_to_events
+
+try:
+    from _pyrepl.console import Event
+    from _pyrepl.windows_console import WindowsConsole
+except ImportError:
+    pass
+
+
+@patch("os.write")
+@unittest.skipIf(sys.platform != "win32", "No Unix event queue on Windows")
```

Comment:
The skip reason is backward for this test class. 
```suggestion
@unittest.skipIf(sys.platform != "win32", "Test class specifically for Windows")
```